src/sass/partials/_about.scss

```diff
@@ -1,27 +1,96 @@
 .about {
-  display: block;
-  flex-direction: row;
-}
-.about__header {
-  font-family: 'Montserrat' sans-serif;
-  font-weight: 700;
-  font-size: 24px;
-  line-height: 29px;
+  display: inline-block;
+
+  &__header {
+    content: '';
+    display: table;
+    width: 100%;
+    margin: 0;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 1.21;
+  }
+  .line {
+    display: table-cell;
+    width: 100%;
+    padding-left: 14px;
+    border: none;
+  }
+  .line:after {
+    content: '';
+    border-top: 1px solid #000000;
+    width: 100%;
+    display: inline-block;
+    vertical-align: middle;
+  }
 
-  width: 91px;
-  height: 29px;
-  left: 0px;
-  top: 0px;
+  &__text {
+    font-family: 'Montserrat', sans-serif;
+    font-size: 16px;
+    line-height: 1.25;
+    color: $black;
+    padding-top: 32px;
+    padding-bottom: 16px;
+  }
+}
+.about-picture {
+  display: flex;
+  width: 100%;
+  height: auto;
 }
+.about-list {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin: 0;
+}
+
+@media screen and (min-width: 768px) and (max-width: 1280px) {
+  .about {
+    display: flex;
+    flex-direction: column;
+    &__header {
+      font-size: 44px;
+      line-height: 1.23;
+    }
+    &__text {
+      margin-right: 54px;
+      font-size: 20px;
+      line-height: 1.2;
+    }
+  }
+  .about-list {
+    flex-direction: row;
+    align-items: center;
+    padding-bottom: 100px;
```

Comment:
Zrób tu raczej margin. Jeśli Ewa też nada na górze sekcji store padding to będzie go już 200, jeśli nie wyjdzie tyle razem containerem.